README.md

```diff
@@ -267,6 +267,21 @@ class TicketFactModel < ActiveReporting::FactModel
 end
 ```
 
+## Configuring Aggregate Expressions
+
+If you need more granular control over any aggregate function (e.g., `SUM`, `COUNT`, `AVG`, etc.), you can declare an [aggregate expression](https://www.postgresql.org/docs/8.2/static/functions-aggregate.html) in the fact model. This use case can arise if you need more granular control over the results of an aggregate function specified in an `ActiveReporting::Metric`.
```

Comment:
Why is this documentation pointing to Postgresql? Is this something specific for this database? According to specs it should work on all supported databases right?

Reply:
It should work on any database. I agree it would be better to provide links both databases.